.vscode/settings.json

```diff
@@ -46,5 +46,7 @@
     "jsonc",
     "yaml"
   ],
-  "testing.automaticallyOpenTestResults": "neverOpen"
+  "testing.automaticallyOpenTestResults": "neverOpen",
+  // Use the project's typescript version
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Why do we need to specify this?

Reply:
This can avoid the situation where the ts version installed in the project is inconsistent with the ts version of vscode, which may cause code errors when using the ts version of vscode.

Reply:
I think this is worth a try. I just had a weird type error on renovate CI https://github.com/vitest-dev/vitest/pull/7600, but I couldn't see it on my vscode. This was because CI updated typescript 5.8 while vscode default was on 5.7.